### PR TITLE
Various fixes

### LINKED ISSUE #889: The cloner biomass can go into the negatives.
I can't be very specific because I don't know how it occured, but the cloner biomass went into the negatives and we essentially had infinite biomass.

http://imgur.com/I02wdIB

### LINKED ISSUE #907: Syndiborg Emag
The Syndiborg's Emag is unable to emag the covers of other borgs, making the Flash>Emag>Crowbar>Emag combo they are meant to use nonexistant.

I do not quite know what the cause of this is, but they do not got the "You emag the cover" message when clicking a borg, while doing so while a organic will. Their Emag works on everything else though.

### PR DESCRIPTION
**Pull request by veganzombeh**
Fixes #907 and #905. Potentially fixes #889, I couldn't reliably reproduce it, but the cloning machine now loses biomass when it starts cloning, rather than when the clone is ejected, which I suspect caused it.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -819,9 +819,6 @@ var/list/robot_verbs_default = list(
 		return ..()
 
 /mob/living/silicon/robot/emag_act(user as mob)
-	if(!ishuman(user))
```
Comment (tigercat2000) [inline code/modules/mob/living/silicon/robot/robot.dm]: it would probably be better to change this to if(!ishuman(user) && !issilicone(user))

Comment (veganzombeh) [inline code/modules/mob/living/silicon/robot/robot.dm]: You're right. For some reason I was thinking they should only be able to unlock covers, not subvert other borgs.

**Comment by Krausus** [comment]
I'm pretty sure #889 is caused by autocloning. Specifically, [this bit of code](https://github.com/ParadiseSS13/Paradise/blob/672305e034fe6eae4beca7f2f3eac2f93f48a798/code/game/machinery/computer/cloning.dm#L35-L39):
```
if(!(pod1.occupant || pod1.mess) && (pod1.efficiency > 5))
	for(var/datum/dna2/record/R in src.records)
		if(!(pod1.occupant || pod1.mess))
			if(pod1.growclone(R))
				records.Remove(R)	
```
`growclone` doesn't check how much biomass is in the cloner. In fact, the *only* thing that ever checks whether there's enough biomass is the cloning console's UI code! And not even that works quite right, since [a typo](https://github.com/ParadiseSS13/Paradise/blob/672305e034fe6eae4beca7f2f3eac2f93f48a798/code/game/machinery/computer/cloning.dm#L147) prevents the Clone button from being properly disabled.

**Comment by veganzombeh** [comment]
Okay, #889 should *actually* be fixed now.

**Pull request merged**